app/models/cert_issue_request.rb

```diff
@@ -29,15 +29,19 @@ class CertIssueRequest
   validate :validate_no_wildcards
 
   def fqdns
-    alt_names + [ common_name ]
+    alt_names_array + [ common_name ]
```

Comment:
.to_a.flatten instead of a method? If you want to keep the method, make it private and write a test perhaps?

Reply:
it is public because used by the `DomainOwnershipService` and there is a test.